fe/fe-core/src/main/java/org/apache/doris/analysis/ExprSubstitutionMap.java

```diff
@@ -39,7 +39,7 @@
  * See Expr.substitute() and related functions for details on the actual substitution.
  */
 public final class ExprSubstitutionMap {
-    private static final Logger LOG = LoggerFactory.getLogger(ExprSubstitutionMap.class);
+    private static final Logger LOG = LogManager.getLogger(SlotRef.class);
```

Comment:
What's the difference between LogManager and LoggerFactory

Reply:
Currently, we all use log4j directly instead of slf4j.
Maybe we should change all of them later.